eng/scripts/vs.18.buildtools.intpreview.json

```diff
@@ -0,0 +1,31 @@
+{
+    "channelUri": "https://aka.ms/vs/18/intpreview/channel",
+    "channelId": "VisualStudio.18.IntPreview",
+    "includeRecommended": false,
+    "addProductLang": [
+        "en-US"
+    ],
+    "add": [
+        "Microsoft.Net.Component.4.6.2.TargetingPack",
+        "Microsoft.Net.Component.4.7.2.SDK",
+        "Microsoft.Net.Component.4.7.2.TargetingPack",
+        "Microsoft.VisualStudio.Component.FSharp.MSBuild",
+        "Microsoft.VisualStudio.Component.NuGet",
+        "Microsoft.VisualStudio.Component.NuGet.BuildTools",
+        "Microsoft.VisualStudio.Component.VC.ATL",
+        "Microsoft.VisualStudio.Component.VC.ATL.ARM64",
+        "Microsoft.VisualStudio.Component.VC.Tools.ARM64",
+        "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+        "Microsoft.VisualStudio.Component.VC.14.29.16.11.ARM64",
```

Comment:
Where did these numbers come from?

Reply:
I think those are just copied from the v17 one https://github.com/dotnet/aspnetcore/blob/main/eng/scripts/vs.17.intpreview.json